system-addon/test/unit/lib/Store.test.js

```diff
@@ -38,55 +49,129 @@ describe("Store", () => {
     store.dispatch({type: "FOO"});
     assert.calledOnce(store._mm.middleware);
   });
-  describe("#init", () => {
-    it("should dispatch an init action", () => {
-      sandbox.spy(store, "dispatch");
+  describe("#initFeed", () => {
+    it("should add an instance of the feed to .feeds", () => {
+      class Foo {}
+      store._feedConstructors = {foo: () => new Foo()};
 
-      store.init();
+      store.initFeed("foo");
 
-      assert.calledWith(store.dispatch, {type: at.INIT});
+      assert.isTrue(store.feeds.has("foo"), "foo is set");
+      assert.instanceOf(store.feeds.get("foo"), Foo);
     });
-    it("should dispatch an init action", () => {
-      sandbox.spy(store, "dispatch");
+    it("should add a .store property to the feed", () => {
+      class Foo {}
+      store._feedConstructors = {foo: () => new Foo()};
```

Comment:
Probably not worth fixing in these tests, but for the future, it seems like we could stick to public APIs by simply creating a new store and adding Foo to calling store.init().  This will make the tests more useful for refactorings.